.github/workflows/pr.yml

```diff
@@ -47,7 +47,7 @@ jobs:
 
       - name: Check bundle size
         # this should match the os and version used in the release.yml
-        if: startsWith(matrix.os, 'ubuntu-20.04') && startsWith(matrix.node, '14.')
```

Comment:
This stops the `check bundle size` run from happening twice; we don't want it to run on `14.0.0`